tests/test_lab_config.py

```diff
@@ -16,3 +20,93 @@ def test_ilab_config_show(cli_runner: CliRunner) -> None:
     assert parsed
 
     assert configuration.Config(**parsed)
+
+
+def test_ilab_config_init_with_env_var_config(
+    cli_runner: CliRunner, tmp_path: pathlib.Path
+) -> None:
+    # create a dummy config file
+    cfg = configuration.get_default_config()
+    assert cfg.general.log_level == "INFO"
+    cfg.general.log_level = "DEBUG"
+    cfg_file = tmp_path / "config.yaml"
+    with cfg_file.open("w") as f:
+        yaml.dump(cfg.model_dump(), f)
+
+    result = cli_runner.invoke(
+        lab.ilab, ["config", "init"], env={"ILAB_GLOBAL_CONFIG": cfg_file.as_posix()}
+    )
+    assert result.exit_code == 0, result.stderr
+
+    # the following feels ugly, if someone has a better way to extract the path from the output
+    pattern = r"`([^`]*)`"
+    matches = re.findall(pattern, result.stdout)
+    assert matches
+    extracted = None
+    for match in matches:
+        if "config.yaml" in match:
+            extracted = match
+            break
+
+    assert (
+        extracted is not None
+    ), "No path containing 'config.yaml' was found in matches"
+
+    # load the generated config file
+    extracted_path = pathlib.Path(extracted)
+    assert extracted_path.exists()
+    with extracted_path.open(encoding="utf-8") as f:
+        parsed = yaml.safe_load(f)
+        assert parsed
+
+        # check that the log level was set to DEBUG
+        assert configuration.Config(**parsed).general.log_level == "DEBUG"
+
+    # remove the file
+    extracted_path.unlink()
+
+
+# TODO: remove this test once the deprecated alias 'ilab init' is removed
+def test_ilab_init_with_env_var_config(
+    cli_runner: CliRunner, tmp_path: pathlib.Path
+) -> None:
+    # create a dummy config file
+    cfg = configuration.get_default_config()
+    assert cfg.general.log_level == "INFO"
+    cfg.general.log_level = "DEBUG"
+    cfg_file = tmp_path / "config.yaml"
+    with cfg_file.open("w") as f:
+        yaml.dump(cfg.model_dump(), f)
+
+    result = cli_runner.invoke(
+        lab.ilab, ["init"], env={"ILAB_GLOBAL_CONFIG": cfg_file.as_posix()}
```

Comment:
so what's the difference between the two cases? This line only? Can we make these into two variants of the same base test case? This is very redundant.